src/main/java/com/diamondfire/helpbot/bot/command/argument/impl/parsing/parser/SingleArgumentParser.java

```diff
@@ -20,10 +20,10 @@ public SingleArgumentParser(SingleArgumentContainer<A> container) {
         Deque<String> rawArgs = args.popStack();
         Argument<A> arg = getContainer().getArgument();
         
-        if (rawArgs.peek() == null) {
+        if (rawArgs.peek() == null && !(arg instanceof FallbackArgument<?>)) {
```

Comment:
We are going to need to change this to fit the current arg system. We shouldn't need to cast.

Reply:
How? A new parser?